### PR TITLE
RAIN-12517 - Update README.md on how to manually upgrade their agent

### PR DESCRIPTION
**Pull request by LizKeeneLW**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,101 @@
 <a href="https://www.lacework.com/"><img src="https://www.lacework.com/wp-content/uploads/2019/07/Lacework_Logo_color_2019.svg" width="250px" height="100px" title="Lacework" alt="Lacework"></a>
 
-# Lacework Agent Release
-The source for officially supported Lacework agent releases
+# Lacework Agent Releases
+This Lacework agent GitHub site provides the officially supported Lacework agent releases. This site provides both the Lacework agent install and source files available for downloading.
```
Comment (rakeshlacework) [inline README.md:4]: there are no sources available for download

File: README.md [hunk 2]
```diff
@@ -1,9 +1,101 @@
...
+Complete the following prerequisites steps before starting any of the agent install procedures below.
+1. Download the release package ```release.tgz``` file from this Lacework GitHub Agent Release repository, where ```release``` is replaced by the agent release number.
+    1. In the right frame, click **Releases** to view all the available agent releases.  
+    2. Find a release and click ```release.tgz``` to download the file that contains the agent installers, where ```release``` is replaced by the agent release number.
+2. Unzip the ```release.tgz``` file into a temporary directory such as ```/tmp```.
+3. Using the Lacework Console, create an agent access token  by following the instructions in [Create Agent Access Tokens](https://support.lacework.com/hc/en-us/articles/360036425594).
+4. In the Lacework Console, copy the Access Token by clicking the **Copy to clipboard icon** icon.
+
+## Install Using the install.sh Script
+For single host installations, Lacework recommends using the installation script called ```install.sh```.
```
Comment (rakeshlacework) [inline README.md:25]: Please includes a link to the support page which lists supported os distro's

File: README.md [hunk 3]
```diff
@@ -1,9 +1,101 @@
...
-# Lacework Agent Release
-The source for officially supported Lacework agent releases
+# Lacework Agent Releases
+This Lacework agent GitHub site provides the officially supported Lacework agent releases. This site provides both the Lacework agent install and source files available for downloading.
 
-# Lacework Agent Documentation
-Install and configure agent, use the polygraph, and learn about the host dossiers <a href="https://support.lacework.com/hc/en-us/categories/360001044834-Lacework-for-Workload-Security" target="_blank">**here**</a>.
+# Releases
+In the right frame, click **Releases** to view all the available agent releases.
+Under the **Assets**, all the release download files are listed. These download files contain the agent installers and the source code. In addition, each release contains a direct link to the specific agent release notes.
```
Comment (rakeshlacework) [inline README.md:8]: please omit source code

File: README.md [hunk 4]
```diff
@@ -1,9 +1,101 @@
...
+   $ tar -xzf chef.tar.gz
+   ```
+5. Copy the ```install.sh``` file with the updated token to the appropriate directory.
+   ```
+   $ cp install.sh /tmp/chef/datacollector/files/default
+   ```
+6. Install the agent by following the instructions in [Install with Chef](https://support.lacework.com/hc/en-us/articles/360005321413).
 
-Latest Lacework Documentation are available <a href="https://support.lacework.com/hc/en-us" target="_blank">**here**</a>.
+## Install using Daemonset Deployment files for Kubernetes
```
Comment (rakeshlacework) [inline README.md:57]: For k8s and docker swarm please point to the docker command to use to pull the release specific image listed on the releases page under Lacework Agent Docker image

File: README.md [hunk 5]
```diff
@@ -1,9 +1,101 @@
...
+
+# Install Agents
+This README.md file provides installation instructions for installing the agent from this Lacework Agent GitHub repository.
+You can also install the agent from the Lacework Console. For a listing of the different agent install methods, see [Agent Install Options](https://support.lacework.com/hc/en-us/sections/360002375733).
+
+## Install Prerequisites
+Complete the following prerequisites steps before starting any of the agent install procedures below.
+1. Download the release package ```release.tgz``` file from this Lacework GitHub Agent Release repository, where ```release``` is replaced by the agent release number.
+    1. In the right frame, click **Releases** to view all the available agent releases.  
+    2. Find a release and click ```release.tgz``` to download the file that contains the agent installers, where ```release``` is replaced by the agent release number.
```
Comment (rakeshlacework) [inline README.md:19]: Please add a section which describes validation of the the release tar images using the checksum.txt for the release. ```md5sum -c checksum.txt```

File: README.md (hunk 1 above)
Comment (LizKeeneLW) [reply README.md:4]: Ok. I took that out. I think it's very confusing for the customer to see **Source code (zip)**. That implies source code. 
image

File: README.md (hunk 3 above)
Comment (LizKeeneLW) [reply README.md:8]: Done

File: README.md (hunk 5 above)
Comment (LizKeeneLW) [reply README.md:19]: Done

File: README.md (hunk 2 above)
Comment (LizKeeneLW) [reply README.md:25]: I added this as the first step in the *Install Prerequisites*.

File: README.md (hunk 4 above)
Comment (LizKeeneLW) [reply README.md:57]: Added the following:
  On the **Releases** page of this site, find the appropriate ```docker pull``` command for a specific release under the **Lacework Agent Docker image** heading.

File: README.md [hunk 6]
```diff
@@ -1,9 +1,116 @@
...
+Under the **Assets**, all the release download files are listed. These download files contain the agent installers. In addition, each release contains a direct link to the specific agent release notes and the ```docker pull``` command for pulling down a Lacework docker install image. 
+For all the agent release notes, see [Release Updates](https://support.lacework.com/hc/en-us/categories/360000539793-Release-Updates).
+
+# Install Agents
+This README.md file provides installation instructions for installing the agent from this Lacework Agent GitHub repository.
+You can also install the agent from the Lacework Console. For a listing of the different agent install methods, see [Agent Install Options](https://support.lacework.com/hc/en-us/sections/360002375733).
+
+## Install Prerequisites
+Complete the following prerequisites steps before starting any of the agent install procedures below.
+1. Verify that the Lacework agent supports the distribution installed your machine. For more information, see [Supported Operating Systems](https://support.lacework.com/hc/en-us/articles/360005230014).
```
Comment (rakeshlacework) [inline README.md:17]: Should we add a note on the version of sed in use for examples - 

```rakesh@ubuntu-16:~/works/src/github.com/lacework/agent$ sed --version
sed (GNU sed) 4.2.2
Copyright (C) 2012 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.```

File: README.md [hunk 7]
```diff
@@ -1,9 +1,116 @@
...
+
+## Install using Docker Swarm Deployment Files
+1. Complete the [prerequisites](#install-prerequisites) steps.
+2. Set the ```token``` environment variable equal to the agent token copied from the Lacework Console.
+   ```
+   $ export token=<copied_agent_access_token>
+   ```
+3. Use ```sed``` to  replace ```LaceworkAccessToken``` with the valid agent access token in the ```docker-compose.yml``` and ```docker-compose-v3.yml``` files.
+   ```
+   $ sed -i.-i.bak "s/\${LaceworkAccessToken}/${token}/g" docker-compose.yml
```
Comment (rakeshlacework) [inline README.md:91]: Please omit -i. from both commands. this should say something like -
```sed -i.bak "s/\${LaceworkAccessToken}/${token}/g" docker-compose.yml```

Comment (LizKeeneLW) [reply README.md]: Good catch. fixed.

File: README.md (hunk 6 above)
Comment (LizKeeneLW) [reply README.md:17]: In the Install Prerequisites, I added "Use sed (GNU sed) 4.2.2 in the procedures below. ".

File: README.md [hunk 8]
```diff
@@ -1,9 +1,117 @@
...
+For all the agent release notes, see [Release Updates](https://support.lacework.com/hc/en-us/categories/360000539793-Release-Updates).
+
+# Install Agents
+This README.md file provides installation instructions for installing the agent from this Lacework Agent GitHub repository.
+You can also install the agent from the Lacework Console. For a listing of the different agent install methods, see [Agent Install Options](https://support.lacework.com/hc/en-us/sections/360002375733).
+
+## Install Prerequisites
+Complete the following prerequisites steps before starting any of the agent install procedures below.
+1. Verify that the Lacework agent supports the distribution installed your machine. For more information, see [Supported Operating Systems](https://support.lacework.com/hc/en-us/articles/360005230014).
+2. Use sed (GNU sed) 4.2.2 in the procedures below.
```
Comment (rakeshlacework) [inline README.md:18]: Would it better to include "or higher version". 

Basically I have tested the instructions to work with that version is assume newer ones would still work.

Comment (LizKeeneLW) [reply README.md]: I changed it to 'Use sed (GNU sed) version 4.2.2 or higher in the procedures below.'

**Pull request merged**